### PR TITLE
Configure tilejson api version dynamically

### PR DESCRIPTION
**Pull request by rmarianski**
@zerebubuth could you review please?

### CODE REVIEW

**Comment by rmarianski** [comment]
Connects to https://github.com/tilezen/vector-datasource/pull/1056

File: tilejson/tilejson.json.erb [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "name" : "<%= @params[:name] %>",
     "private" : true,
     "scheme" : "xyz",
-    "tilejson" : "2.0.0",
+    "tilejson" : "<%= @params[:tilejson_version] %>",
     "tiles" : [
         "http://<%= node[:tilejson][:host] %>/mapzen/vector/<%= node[:tilejson][:vector_tiles_version] %>/all/{z}/{x}/{y}.<%= @params[:extension] %>?api_key={api_key}"
```
Comment (zerebubuth) [inline tilejson/tilejson.json.erb]: If this is related to mapzen/chef-mapzen_tilequeue#27 then does `:vector_tiles_version` need to change to `:url_path_version`?

Comment (rmarianski) [inline tilejson/tilejson.json.erb]: Nice catch, updated in b18f1b9.

I also noticed that I was referencing the api version incorrectly too. Should be fixed in 82aa063.

Comment (zerebubuth) [inline tilejson/tilejson.json.erb]: Does that maens the `:tilejson_version` on L20 also needs to come from `node[:tilejson]`?

Comment (rmarianski) [inline tilejson/tilejson.json.erb]: Yes ;) Thanks for catching that one too. Fixed in 9481e50.

**Pull request merged**